### PR TITLE
SUBMARINE-263. Document grammar improvement

### PR DESCRIPTION
**Pull request by hhhizzz**
### What is this PR for?
change some words and sentences, make the document better.

### What type of PR is it?
Improvement

### What is the Jira issue?
https://issues.apache.org/jira/projects/SUBMARINE/issues/SUBMARINE-263

### How should this be tested?
https://travis-ci.org/hhhizzz/hadoop-submarine/builds/603971718

### Questions:
* Does the licenses files need updates? Yes/No
* Are there breaking changes for older versions? Yes/No
* Does this needs documentation? Yes/No

### CODE REVIEW

File: docs/community/README.md [hunk 1]
```diff
@@ -50,13 +50,13 @@ Contributions are welcomed and greatly appreciated. See [CONTRIBUTING.md](contri
 
 ## How Do I Become a Committer?
 
-First of all you need to get involved and be a Contributor.
+First of all, you need to get involved and be a Contributor.
 
-Based on your track-record as a contributor, Per Apache code, PMCs vote on committership, may invite you to be a committer (after we've called a vote). When that happens, if you accept, the following process kicks into place...
+Based on your track-record as a contributor, Per Apache code, PMCs vote on committers, may invite you to be a committer (after we've called a vote). When that happens, if you accept, the following process kicks into place...
```
Comment (jojochuang) [inline docs/community/README.md]: actually, the word "comittership" makes more sense here.

Comment (hhhizzz) [reply docs/community/README.md]: Got it

File: docs/README.md [hunk 2]
```diff
@@ -67,12 +67,12 @@ Docker related dependencies, including:
...
 - Docker network which allows Docker container can talk to each other across different nodes.
 
-And when GPU plan to be used:
+And when GPU plans to be used:
 
 - GPU Driver.
 - Nvidia-docker.
 
-For your convenience, we provided installation documents to help you to setup your environment. You can always choose to have them installed in your own way.
+For your convenience, we provided installation documents to help you to set up your environment. You can always choose to have them installed in your own way.
```
Comment (yuanzac) [inline docs/README.md]: provided needs to be changed to provide

File: docs/design/SubmarineClusterServer.md [hunk 3]
```diff
@@ -57,8 +57,8 @@ some programs may appear abnormal. Of course, we also detected this in the syste
 
 ### Universal design
 
-Modular design, because Submarine (Workbench) Server exists in Submarine system, these two services need to provide clustering capabilities, 
-so we abstract the cluster function into a separate module for development, so that Submarine (Workbench) Server can reuse the cluster function module.
+Modular design, because Submarine (Workbench) Server exists in the Submarine system, these two services need to provide clustering capabilities,
```
Comment (yuanzac) [inline docs/design/SubmarineClusterServer.md]: We can use either because or so, not both of them

**Comment by jojochuang** [comment]
+1 pending Travis CI

**Comment by yuanzac** [comment]
+ 1
Thanks @hhhizzz and @jojochuang.

**Pull request closed without merging**